### ISSUE TITLE
Smooth FieldToParticleInterpolation for Field Ionization

### ISSUE THREAD

**Issue by n01r** [ComputationalRadiationPhysics/picongpu#2001]
It seems that for certain setups the "noise" created by local electric fields of free electrons is enough to ionize atoms/ions in an unphysical way. 

The proposition stands that smoothing the used `FieldToParticleInterpolation` to more physical scales (larger than the Debye length) will reduce the noise and produce results of better quality.

**Comment by ax3l** [comment]
Thanks for already opening an issue (but please keep the scientific practice of quoting ideas that are not your own).

To be more precise, it's not about the Debye lengths but about smoothing to a scale between the cell size (arbitrary, non-physical, not necessarily in the order of the Debye length) and the smallest resolved wavelength.

In the case we are looking in (solid target) this is a smoothing to better account for the effects of plasma waves (while cutting their top amplitude a little) and neglecting small ringing from in-cell noise.

**Comment by n01r** [comment]
You're right - sorry - I did not intend to claim it. I wanted to note it down before I left.
